src/component/notification.js

```diff
@@ -37,19 +40,21 @@ const barStyles = StyleSheet.create({
 
 export const NotificationBar = ({ notification, display, style }) =>
   notification && display ? (
-    <View style={[barStyles.bar, style]}>
-      <View style={barStyles.msgWrapper}>
-        <Alert type={notification.type} style={barStyles.alert} />
-        <Text style={barStyles.text}>{notification.message}</Text>
+    <SafeAreaView style={barStyles.safe}>
```

Comment:
IMO it looks a bit weird/abrupt to have the rest of the view shift downward when a notification appears.